condenser/query.py

```diff
@@ -104,6 +104,32 @@ def as_structarray(self):
 
         return arr
 
+    def with_transformed_geometries(self, target_srid):
+        """Transform all SRID-aware columns to given target SRID
+
+        Args:
+          target_srid (int): The SRID to reproject the geometries into.
+
+        See also:
+          Columns that are not SRID-aware do not support coordinate
+          transformations. If an SRID is known from metadata, use a function
+          appropriate to your database backend to set it (e.g. `ST_SetSRID`
+          for PostGIS and `setsrid` for SQLite/spatialite)
+        """
+        if not has_geo:
+            raise ImportError("This function requires GeoAlchemy2 and pygeos")
+        srid = int(target_srid)
+        new_columns = []
+        for descr in self.column_descriptions:
+            if descr["type"].__class__ == Geometry and descr["type"].srid > 0:
+                new_columns.append(
+                    ST_Transform(descr["expr"], srid).label(descr["name"])
+                )
+            else:
+                new_columns.append(descr["expr"])
```

Comment:
Seems to be missing `.label(descr["name"])`

Reply:
No because there is no change here to the expression, the original label is maintained.